test/component/executors/testExecutorsInvite.js

```diff
@@ -24,8 +24,22 @@ describe('executors-invite', () => {
 
     describe('Verify Content, Errors and Redirection', () => {
 
-        it('test content loaded on the page', (done) => {
-            testWrapper.testContent(done);
+        it('test correct content loaded on the page when only 1 other executor', (done) => {
+            sessionData.executors.executorsNumber = 2;
+            testWrapper.agent.post('/prepare-session/form')
+                .send(sessionData)
+                .end(() => {
+                    testWrapper.testContent(done, ['heading3Multiple']);
+                });
+        });
+
+        it('test correct content loaded on the page when more than 1 other executor', (done) => {
+            sessionData.executors.executorsNumber = 3;
```

Comment:
We don't ned to set this here as it's already set in the test data in `test/data/executors-invites.json`